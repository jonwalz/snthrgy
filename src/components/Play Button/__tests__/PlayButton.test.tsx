import { render } from "@testing-library/react";
import React from "react";
import { PlayButton } from "..";

describe("play button", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<PlayButton />);
    expect(asFragment()).toMatchSnapshot();
  });
});
