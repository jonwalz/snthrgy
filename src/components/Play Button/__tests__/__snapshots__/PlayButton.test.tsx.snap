// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`play button renders correctly 1`] = `
<DocumentFragment>
  <button
    class="sc-gsTCUz eLfczQ"
  >
    <div
      class="sc-bdfBwQ hyqfeI"
    />
  </button>
</DocumentFragment>
`;
